### PR TITLE
+logo logic to support url for http/s strings

### PR DESCRIPTION
**Pull request by raoulmahtani**

### CODE REVIEW

**Comment by raoulmahtani** [comment]
#232 #231

File: openvpn-monitor.py [hunk 1]
```diff
@@ -625,7 +625,10 @@ def print_html_header(self):
         if self.logo:
             output('<a href="#" class="pull-right"><img alt="Logo" ')
             output('style="max-height:46px; padding-top:3px;" ')
-            output('src="images/{0!s}"></a>'.format(self.logo))
+            if "http://" in self.logo or "https://" in self.logo:
```
Comment (furlongm) [inline openvpn-monitor.py:628]: Could we use `startswith` instead of `in`? Otherwise, this looks good.

Comment (raoulmahtani) [reply openvpn-monitor.py]: updated!

**Review by furlongm — APPROVED** [review]

**Comment by raoulmahtani** [comment]
hey @furlongm, dont think I have rights to merge,  can you assist?

**Pull request merged**